contribs/gmf/src/sass/vars_only.scss

```diff
@@ -34,13 +34,19 @@ $above-all: 9999;
 
 $nav-width: 20rem !default;
 
-$border-radius-base: 2px !default;
-$border-radius-small: 1px !default;
+$border-radius-base: 0 !default;
+$border-radius-small: 0 !default;
```

Comment:
Shouldn't we remove them and use `$border-radius-sm` and `$border-radius`

Reply:
+1

Reply:
In fact, I prefer to use our own variable ($border-radius-base) So If customers want a radius with bootstrap thing they can have it. And using our own variable, we are not dependant of boostrap as well.

We never use $border-radius-small, I remove it.

Reply:
OK :-)